ci/check-style.sh

```diff
@@ -61,57 +50,29 @@ replace_original_if_changed() {
 
 # Apply cmake_format to all the CMake list files.
 #     https://github.com/cheshirekow/cmake_format
-find . \( "${ignore[@]}" \) -prune -o \
-  \( -name 'CMakeLists.txt' -o -name '*.cmake' \) \
-  -print0 |
+git ls-files -z | grep -zE '((^|/)CMakeLists\.txt|\.cmake)$' |
   xargs -0 cmake-format -i
 
 # Apply clang-format(1) to fix whitespace and other formatting rules.
 # The version of clang-format is important, different versions have slightly
 # different formatting output (sigh).
-find google/cloud \( -name '*.cc' -o -name '*.h' \) -print0 |
-  xargs -0 clang-format -i
-
-# Apply several transformations that cannot be enforced by clang-format:
-#     - Replace any #include for grpc++/* with grpcpp/*. The paths with grpc++
-#       are obsoleted by the gRPC team, so we should not use them in our code.
-#     - Replace grpc::<BLAH> with grpc::StatusCode::<BLAH>, the aliases in the
-#       `grpc::` namespace do not exist inside google.
-find google/cloud \( -name '*.cc' -o -name '*.h' \) -print0 |
-  while IFS= read -r -d $'\0' file; do
-    # We used to run run `sed -i` to apply these changes, but that touches the
-    # files even if there are no changes applied, forcing a rebuild each time.
-    # So we first apply the change to a temporary file, and replace the original
-    # only if something changed.
-    sed -e 's/grpc::\([A-Z][A-Z_][A-Z_]*\)/grpc::StatusCode::\1/g' \
-      -e 's;#include <grpc\\+\\+/grpc\+\+.h>;#include <grpcpp/grpcpp.h>;' \
-      -e 's;#include <grpc\\+\\+/;#include <grpcpp/;' \
-      "${file}" >"${file}.tmp"
-    replace_original_if_changed "${file}" "${file}.tmp"
-  done
+git ls-files -z | grep -zE '\.(cc|h)$' | xargs -0 clang-format -i
 
 # Apply buildifier to fix the BUILD and .bzl formatting rules.
 #    https://github.com/bazelbuild/buildtools/tree/master/buildifier
-find . \( "${ignore[@]}" \) -prune -o \
-  \( -name WORKSPACE -o -name BUILD \
-  -o -name '*.BUILD' -o -name '*.bzl' \) \
-  -print0 |
+git ls-files -z | grep -zE '\.(BUILD|bzl)$' | xargs -0 buildifier -mode=fix
+git ls-files -z | grep -zE '((^|/)BUILD|WORKSPACE)$' |
```

Comment:
It looks like the `(^|/)` should be outside the parentheses.

Reply:
Done